common/src/java/org/apache/hadoop/hive/common/FileUtils.java

```diff
@@ -1387,6 +1388,47 @@ public static RemoteIterator<LocatedFileStatus> listFiles(FileSystem fs, Path pa
         status -> filter.accept(status.getPath()));
   }
 
+  /**
+   * Resolves a symlink on a local filesystem. In case of any exceptions or scheme other than "file"
+   * it simply returns the original path. Refer to DEBUG level logs for further details.
+   * @param path
+   * @param conf
+   * @return
+   * @throws IOException
```

Comment:
can you fill these values? returns resolved path and something like that

Reply:
will do